etc/travis/suites/application/script/validate-composer-dependencies

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../../../bash/common.lib.sh"
+
+print_header "Validating (Composer require checker)" "Sylius"
+run_command "(composer global require maglnet/composer-require-checker && $HOME/.composer/vendor/bin/composer-require-checker --config-file=composer-require-checker.json)"
```

Comment:
Shouldn't `composer global require `maglnet/composer-require-checker` be part of some install step with the possibility to add this library to cache?

Reply:
I'd vote for downloading the PHAR and caching it (it's the recommended way by the package itself). We do the same with Selenium now.